src/sql/_testing.py

```diff
@@ -187,6 +188,9 @@ def database_ready(
             eng.close()
             print(f"{database} is initialized successfully")
             return True
+        except ModuleNotFoundError as e:
+            msg = "Please install " + e.name.lower() + " first!"
```

Comment:
please use string format instead of +

why lowercase the name?

Reply:
Yeah lowercase is not nessaray

Also the string format is called f-string in python, you may check how to include the variable in the string